pandas/tests/frame/test_duplicates.py

```diff
@@ -263,6 +263,30 @@ def test_drop_duplicates_tuple():
     tm.assert_frame_equal(result, expected)
 
 
+def test_drop_duplicates_empty():
+    # GH 20516
+    expected = DataFrame()
+    result = expected.drop_duplicates()
+    tm.assert_frame_equal(result, expected)
+
+    expected = DataFrame(columns=[])
+    result = expected.drop_duplicates()
+    tm.assert_frame_equal(result, expected)
+
+    df = DataFrame(columns=['A', 'B', 'C'])
+    result = df.drop_duplicates()
+    expected = DataFrame(columns=[])  # The column infos are not carrying over
```

Comment:
I don't find this comment useful, rather put a comment about testing with empty columns, and below about an empty index